e2e/test/scenarios/question/reproductions/17514-ui-overlay.cy.spec.js

```diff
@@ -133,9 +133,8 @@ describe("issue 17514", () => {
       // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
       cy.findByText("Save").click();
 
-      cy.get(".Modal").within(() => {
-        cy.button("Save").click();
-      });
+      cy.get(".Modal").button("Save").click();
+      cy.get(".Modal").should("not.exist");
```

Comment:
All failures were happening because a save modal takes forever to animate and was (in some cases) still present in the DOM when Cypress wanted to click on a next element.

This change makes sure that the modal is removed from the DOM first.